frontend/public/components/operator-hub/operator-hub-page.tsx

```diff
@@ -18,59 +18,36 @@ const normalizePackageManifests = (packageManifests: PackageManifestKind[] = [],
     return !packageManifest.status.removedFromBrokerCatalog;
   });
   return _.map(activePackageManifests, packageManifest => {
-    const name = packageManifest.metadata.name;
-    const uid = `${name}/${packageManifest.status.catalogSourceNamespace}`;
-    const defaultIconClass = 'fa fa-clone';
-    const iconObj = _.get(packageManifest, 'status.channels[0].currentCSVDesc.icon[0]');
-    const imgUrl = iconObj ? `data:${iconObj.mediatype};base64,${iconObj.base64data}` : operatorImg;
-    const iconClass = imgUrl ? null : defaultIconClass;
-    const provider = _.get(packageManifest, 'metadata.labels.provider');
-    const tags = packageManifest.metadata.tags;
-    const version = _.get(packageManifest, 'status.channels[0].currentCSVDesc.version');
-    const currentCSVAnnotations = _.get(packageManifest, 'status.channels[0].currentCSVDesc.annotations', {});
+    const currentCSVDesc = _.get(packageManifest, 'status.channels[0].currentCSVDesc', {});
+    const currentCSVAnnotations = _.get(currentCSVDesc, 'annotations', {});
+    const iconObj = _.get(currentCSVDesc, 'icon[0]');
     const installed = (subscriptions || []).some(sub => sub.spec.name === _.get(packageManifest, 'status.packageName'));
 
-    let {
-      description,
-      certifiedLevel,
-      healthIndex,
-      repository,
-      containerImage,
-      createdAt,
-      support,
-      longDescription,
-      categories,
-    } = currentCSVAnnotations;
-    const categoryArray = categories && _.map(categories.split(','), category => category.trim());
-    description = description || _.get(packageManifest, 'status.channels[0].currentCSVDesc.description');
-    longDescription = longDescription || _.get(packageManifest, 'status.channels[0].currentCSVDesc.description');
-    const catalogSource = _.get(packageManifest, 'status.catalogSource');
-    const catalogSourceNamespace = _.get(packageManifest, 'status.catalogSourceNamespace');
-    const providerType = getOperatorProviderType(packageManifest);
     return {
       obj: packageManifest,
       kind: PackageManifestModel.kind,
-      name,
-      uid,
+      name: _.get(currentCSVDesc, 'displayName', packageManifest.metadata.name),
+      uid: `${packageManifest.metadata.name}/${packageManifest.status.catalogSourceNamespace}`,
       installed,
       installState: installed ? 'Installed' : 'Not Installed',
-      iconClass,
-      imgUrl,
-      description,
-      provider,
-      providerType,
-      tags,
-      version,
-      certifiedLevel,
-      healthIndex,
-      repository,
-      containerImage,
-      createdAt,
-      support,
-      longDescription,
-      categories: categoryArray,
-      catalogSource,
-      catalogSourceNamespace,
+      imgUrl: iconObj ? `data:${iconObj.mediatype};base64,${iconObj.base64data}` : operatorImg,
+      description: currentCSVAnnotations.description || currentCSVDesc.description,
+      longDescription: currentCSVDesc.description || currentCSVAnnotations.description,
```

Comment:
It's unclear to my why in one place we prefer `currentCSVAnnotations` and in other we prefer `currentCSVDesc`

Reply:
@galletti94, I'm pulling data from where I can get it. Is there an explanation of these fields and why the short description is in the annotations and the long is in the CSV desc?

Reply:
The way I see it is anything in the CSV desc are fields that are well defined in the CSV crd and are expected to be there when someone writes a CSV. Annotations is an additional field for anything extra that we need for pretty rendering or metadata. The hope is to one day have everything part of the former